pkg/operation/botanist/operatingsystemconfig.go

```diff
@@ -330,7 +333,6 @@ func (b *Botanist) deployOperatingSystemConfigsForWorker(ctx context.Context, ma
 		workerConfig["cri"] = map[string]interface{}{
 			"name": worker.CRI.Name,
 		}
-		downloaderConfig["cri"] = workerConfig["cri"]
```

Comment:
Hm, not sure why this change happened, the line was added as part of the recently merged #2421 and should not be deleted as part of this PR. Can you please revert this line and rebase again to master?

Reply:
strange, I will fix it...